### PR TITLE
create function audio.fastSeek on backspace and advance

### PR DESCRIPTION
**Pull request by jeremyjorand**

### CODE REVIEW

**Review by ThomasDCW — DISMISSED** [review]

File: frontend/src/components/AudioPlayerLoading/index.jsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ export default function AudioPlayerLoading() {
   const [playOrPauseImg, setPlayOrPauseImg] = useState(playImg);
 
   useEffect(() => {
-    axios.get("http://localhost:5000/lessons/:id").then(({ data }) => {
+    axios.get("http://localhost:5000/lessons").then(({ data }) => {
```
Comment (LuigiMenez) [inline frontend/src/components/AudioPlayerLoading/index.jsx:21]: Use .env variable for call of axios
import.meta.env.VITE_BACKEND_URL

**Review by ThomasDCW — APPROVED** [review]

**Review by LuigiMenez — DISMISSED** [review]
Ok for me.

File: frontend/src/components/AudioPlayerLoading/index.jsx [hunk 2]
```diff
@@ -40,6 +40,7 @@ export default function AudioPlayerLoading() {
     } else {
       setTimer(timer + 10);
       setDuration(duration - 10);
+      audio.fastSeek(duration + 10);
```
Comment (LoicBrassart) [inline frontend/src/components/AudioPlayerLoading/index.jsx:43]: We saw that this doesn't work on Chrome. Are you sure you want to push it ?

**Review by LoicBrassart — APPROVED** [review]
To me that's a poor idea, but do as you please. I want isofeature on Firefox/Chrome though or the US is invalid

**Review by ThomasDCW — APPROVED** [review]

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LoicBrassart — APPROVED** [review]
Sorry, meant to approve this PR yesterday
But tomorrow I'll ask you to demo on Chrome anyway :p

**Pull request merged**